be/test/storage/rowset/bitshuffle_page_test.cpp

```diff
@@ -99,7 +99,13 @@ class BitShufflePageTest : public testing::Test {
         Status status = page_decoder.init();
         ASSERT_TRUE(status.ok());
         ASSERT_EQ(0, page_decoder.current_index());
-
+        for (uint i = 0; i < size; i++) {
+            CppType out;
+            page_decoder.at_index(i, &out);
+            if (src[i] != out) {
+                FAIL() << "Fail at index " << i << " inserted=" << src[i] << " got=" << out;
```

Comment:
Why not use ASSERT?

Reply:
ASSERT is OK. The other encoder UT use FAIL(), so use FAIL() at here also